src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1781,7 +1781,7 @@ private string TrimWhiteSpaceHelper(TrimType trimType)
             // Trim specified characters.
             if ((trimType & TrimType.Head) != 0)
             {
-                for (start = 0; start < Length; start++)
+                for (; start < Length; start++)
```

Comment:
Do the same for `end` to make the code symmetric?

Reply:
I meant change `for (end = Length - 1; end >= start; end--)` below to `for (; end >= start; end--)`